src/testbase.rs

```diff
@@ -171,6 +179,49 @@ impl BHShape for Triangle {
     }
 }
 
+impl FromRawVertex for Triangle {
+    fn process(vertices: Vec<(f32, f32, f32, f32)>,
+               _: Vec<(f32, f32, f32)>,
+               polygons: Vec<Polygon>)
+               -> ObjResult<(Vec<Self>, Vec<u16>)> {
+        // Convert the vertices to `Point3`s.
+        let points = vertices
+            .into_iter()
+            .map(|v| Point3::new(v.0, v.1, v.2))
+            .collect::<Vec<_>>();
+
+        // Estimate for the number of triangles, assuming that each polygon is a triangle.
```

Comment:
the following block doesn't only count the triangles. This comment is misleading.